packages/react-scripts/scripts/start.js

```diff
@@ -136,7 +136,12 @@ checkBrowsers(paths.appPath, isInteractive)
       }
 
       console.log(chalk.cyan('Starting the development server...\n'));
-      openBrowser(urls.localUrlForBrowser);
+      if (process.env.NO_PLAYER) return;
+
+      openPlayer(urls.localUrlForBrowser).catch(() => {
+          devServer.close();
+          process.exit();
+      });
```

Comment:
```suggestion
      openPlayer(urls.localUrlForBrowser)
      .then(_ => console.log('Successfully started Player.exe...'))
      .catch(() => {
          devServer.close();
          process.exit();
      });
```

Maybe add a .then handler with a success message?

Reply:
The way it's set up, if the compilation passes it will clear the console, so actually we won't be able to see any log that the Player was opened successfully :(